src/common.tests/JsonHelperTests.cs

```diff
@@ -61,24 +55,6 @@ public void SerializeObjectWithReferenceLoopDefaultSettings()
             Assert.Equal(expected, JsonHelpers.SerializeObject(p2));
         }
 
-        [Fact]
-        public void SerializeObjectWithReferenceLoopCustomSettings()
-        {
-            Person p1 = new Person()
-            {
-                Name = "Frodo"
-            };
-
-            Person p2 = new Person()
-            {
-                Name = "Samwise",
-                BestFriend = p1
-            };
-
-            p1.BestFriend = p2;
-            Assert.Throws<JsonException>(() => JsonHelpers.SerializeObject(p2, serializerSettings));
-        }
-
```

Comment:
Can we add new UT's for the new methods in JsonHelper class please ? ex: ```SerializeForLoggingPurposeIndented, SerializeForLoggingPurpose``` ?

Reply:
Added new tests for more senarios :)